CHANGELOG.md

```diff
@@ -10,6 +10,8 @@ x.x.x Release notes (yyyy-MM-dd)
 * Add a `{RLM}SyncUser.isAdmin` property indicating whether a user is a Realm
   Object Server administrator.
 * Speed up case-insensitive queries on indexed string properties.
+* Add RLMResults's collection aggregate methods to RLMArray.
```

Comment:
RLMResults is used as a proper noun that happens to end in s and not as a plural, so 's is correct.